beets/autotag/match.py

```diff
@@ -241,8 +241,10 @@ def distance(items, album_info, mapping):
         dist.tracks[track] = track_distance(item, track, album_info.va)
         dist.add('tracks', dist.tracks[track].distance)
 
+    count_ign = len([x.ignorable for x in album_info.tracks if x.ignorable])
```

Comment:
Two really tiny comments: first, this is a slightly clearer way to write this.

```py
sum([x.ignorable for x in album_info.tracks])
```

Second, can you please move this under the `# Missing tracks.` comment, because it's related to that code?